src/tests/create_slideshow.js

```diff
@@ -83,17 +94,39 @@ mocha.describe('mathsnuggets', function () {
   })
 
   mocha.it('testing a widget', async function () {
+    await page.goto('http://localhost:5000/slideshow_builder')
     const data = await fetch('http://localhost:5000/api/tests', { method: 'GET' }).then(r => r.json())
     for (const widget in data) {
       const fields = data[widget]
       await selectWidget(page, widget)
       let count = 1
       for (const fieldName in fields) {
         const value = fields[fieldName]
-        await fillInField(page, fieldName, value, fields.length > count ? 'Tab' : 'Enter')
+        await fillInField(page, `span[name="${fieldName}"] textarea`, value, fields.length > count ? 'Tab' : 'Enter')
         count++
       }
       await waitForComputedFields(page)
     }
   })
+
+  mocha.it('test the login', async function () {
+    await page.goto('http://localhost:5000')
+    await clickElement(page, 'a[href="/login"]')
+    await fillInField(page, 'input[name="email"]', 'test@test.com')
+    await fillInField(page, 'input[name="password"]', '12345678')
+    await clickElement(page, 'button.button.is-fullwidth')
+    await clickElement(page, 'button.logout')
+  })
+
+  mocha.it('create a slideshow', async function () {
+    await page.goto('http://localhost:5000')
+    await login(page)
+    await clickElement(page, 'a[href="/resources"]')
+    await clickElement(page, '.panel-block button.is-primary')
+    await fillInField(page, '.modal input[name="title"]', 'New Slideshow')
+    await clickElement(page, '.modal .is-success')
+    await clickElement(page, 'a[href="/resources/new-slideshow"]')
+    await clickElement(page, 'a[href="/resources"]')
```

Comment:
This link doesn't exist in the new version anymore. Either add it back in Slide.vue, or think of another test.